README.md

```diff
@@ -23,7 +23,7 @@ This plugin is optimized for flat config usage (eslint >= 9.0). See [here](https
 import oxlint from "eslint-plugin-oxlint";
 export default [
   ...// other plugins
-  oxlint, // oxlint should be the last one
+  oxlint.configs["flat/recommended"], // oxlint should be the last one
```

Comment:
This is a breaking changes and I think we should avoid using it like this. We support flat config first, legacy config second.

Reply:
I respect your choice as the maintainer. You can close this pr and revert the prior change.

I'll go and fork this as it is useless as is (as we have to use a json eslintrc, legacy config, for oxlint to be able to parse it). I don't see the point of supporting new config.

Thanks

Reply:
I would prefer to contribute to this package, of course. I'm happy to discuss it further:
- a breaking would only mean a minor bump as we are pre-1.0
- semver would instruct users to be cautious of such a bump
- it would be a way for you to introduce a CHANGELOG (maybe manual now, and automatic in the future -- I can help on this too)

Reply:
Thank you for your efforts in this regard! I agree with you.